projects/packages/my-jetpack/_inc/state/selectors.js

```diff
@@ -55,7 +55,7 @@ const productSelectors = {
 
 const purchasesSelectors = {
 	getPurchases: state => state.purchases?.items || [],
-	isRequestingPurchases: state => state.isRequestingPurchases || false,
+	isRequestingPurchases: state => state.purchases?.isFetching || false,
```

Comment:
The tests are failing on this change. But from what I could tell, the prior behavior never worked, which is why I made the change.

Reply:
@retrofox would you be able to advise on this?